recipes/selene-sdk/meta.yaml

```diff
@@ -0,0 +1,45 @@
+package:
+  name: selene-sdk
+  version: 0.0.1 
+
+source:
+  url: https://github.com/FunctionLab/selene/archive/0.0.1.tar.gz
+  sha256: 58853367a6fd6fdd67c3f93592f3d691432cb4fff440cac43b15ac94b15f2eed 
+
+build:
+  number: 0
+  skip: True  # [py27 or win or osx]
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python  # [py3k]
```

Comment:
since you are already skipping py27 on the build section, you don't really need this selector here or on the run section.